Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "Reexport"
+uuid = "189a3867-3050-52da-a836-e630ba90ab69"
+authors = ["Simon Kornblith <simon@simonster.com>"]
+version = "0.2.0"
```

Comment:
Please remove the version; it serves no real purpose and is annoying to keep in sync with the tagged version.